chaos-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: chaos-exporter
+    release: prometheus-stack
+    {{- include "chaos-exporter.labels" . | nindent 4 }}
+  name: chaos-exporter
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: chaos-exporter
+      release: prometheus-stack
+      {{- include "chaos-exporter.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        app: chaos-exporter
+        release: prometheus-stack
+        {{- include "chaos-exporter.selectorLabels" . | nindent 8 }}
+    spec:
+    {{- with .Values.image.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{ end }}
+      containers:
+        - image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          name: chaos-exporter
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          ports:
+            - containerPort: 8080
+              name: tcp
+          env:
+            - name: TSDB_SCRAPE_INTERVAL
+              value: '10'
+            - name: INJECTION_TIME_FILTER
```

Comment:
This env can be taken from values.yml? Have seen some usecase of `INJECTION_TIME_FILTER` in past 😄